### DISCUSSION TITLE
what is the 'parentNo' input mean for count object by type method

### DISCUSSION THREAD

**Discussion by PSARFI** [Dlubal-Software/Dlubal_CSharp_Client#34] [Q&A] [answered]
Hi,

I would like to know what does the second input parameter, which is for 'parent number' mean in this method:

ModelClient.get_object_count(object_types.E_OBJECT_TYPE_MEMBER_ECCENTRICITY,  ????);

Thank you for your help.

**Comment by Rebecca-Coding21** [comment, marked as answer]
Hi @PSARFI,
in case of base objects like member eccentricities there is no parent class. Therefore the parent_no can be set to 0 in your case:
`ModelClient.get_object_count(object_types.E_OBJECT_TYPE_MEMBER_ECCENTRICITY,  0).`
However e.g. for loads the parent_no is important to define the corresponding load case number (parent_no = loadcase number).
